kvbc/include/kvbc_adapter/v4blockchain/blocks_reader_adapter.hpp

```diff
@@ -72,6 +72,13 @@ class BlocksReaderAdapter : public IReader {
 
   // Get the last block ID in the system.
   BlockId getLastBlockId() const override final { return kvbc_->getLastReachableBlockId(); }
+
+  // Get the value of 'key' in `category_id` as it is in 'snapshot_version'
```

Comment:
I think it's enough to have the description only in the db_interfaces and v4_blockchain files

Reply:
OK np.